rendering/cases/reproj-image/main.js

```diff
@@ -0,0 +1,29 @@
+import Map from '../../../src/ol/Map.js';
+import View from '../../../src/ol/View.js';
+import Static from '../../../src/ol/source/ImageStatic.js';
+import {get as getProjection, transform} from '../../../src/ol/proj.js';
+import ImageLayer from '../../../src/ol/layer/Image.js';
+
+const extent = [-123, 37, -122, 38];
+
+const source = new Static({
+  url: '/data/tiles/osm/5/5/12.png',
+  imageExtent: extent,
+  projection: getProjection('EPSG:4326')
```

Comment:
Acutally this should be the other way around, because the tiles are in EPSG:3857. You can configure the view to be in EPSG:4326 instead.